tensorflow_encrypted/protocol/pond.py

```diff
@@ -272,16 +272,16 @@ def helper(x: 'PondPrivateTensor'):
                 if isinstance(xs, PondPrivateTensor):
                     # single input -> single output
                     x = xs
-                    op = receiver.receive_output(helper(x))
+                    t = receiver.receive_output(helper(x))
```

Comment:
`op` is better: `receive_output` doesn't return a tensor, it returns an operation to call when the output should be pushed to the player

Reply:
see comments below; signature of `receive_output` was changed recently but shouldn't have been